Create ui-c8y-1023-4-8-info-dashboard-is-not-loading-with-user-which-do-not-have.md (#3956)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1023-4-8-info-dashboard-is-not-loading-with-user-which-do-not-have.md

---
date: ""

> title: Info dashboard is not loading with user which do not have global read permissions

Reviewer: ```suggestion
title: Info dashboard fails to load for users without global permissions
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5059
version: 1023.4.8
---

> Co-authored-by: Bozhidar Dedov <95280800+bozhidardedov@users.noreply.github.com>

Reviewer: ```suggestion
In **Device management** some **Info** dashboards failed to load for users without global inventory access (i.e. inventory roles only). This issue is now fixed and device **Info** dashboard is available for all users with access to the device.
```

> In **Device management** some **Info** dashboards failed to load for users without global inventory access (i.e. inventory roles only). This issue is now fixed and device **Info** dashboard is available for all users with access to the device.

Reviewer 2: ```suggestion
In the Device Management application, in certain cases dashboards on the **Info** tab failed to load for users without global inventory access (that is, with inventory roles only). This issue is now fixed,  and device dashboards are available for all users with access to the device.
```
